src/analyzing/AST_to_IL.ml

```diff
@@ -1108,6 +1108,11 @@ and stmt_expr env ?e_gen st =
   | G.Return (t, eorig, _) ->
       mk_s (Return (t, expr_opt env eorig)) |> add_stmt env;
       expr_opt env None
+  | G.DefStmt (ent, G.VarDef { G.vinit = Some e; vtype = _typTODO }) ->
```

Comment:
Maybe a comment pointing out that we end up here due to `Elixir_to_elixir`, or do we know this happens also with Ruby or Julia ?

Reply:
I tried to see if this happens in Ruby or Julia, but couldn't make this pattern occur. I'll add a comment about Elixir. The language guard would probably just apply to Elixir for now too.